src/AppKit/NSFont.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.CoreGraphics;
+using XamCore.CoreAnimation;
+
+using CGGlyph = System.UInt16;
+
+namespace XamCore.AppKit {
+	public partial class NSFont
+	{
+		[Mac (10,13)]
+		public unsafe void GetBoundingRects (CGRect [] bounds, CGGlyph [] glyphs, nuint glyphCount)
```

Comment:
`Get` without a return value ? or `out` parameters ?
Maybe we can expose something better ? or at least rename the parameters to make it clear what is in and out